manager/filters.py

```diff
@@ -44,7 +44,7 @@ def _filter_cve_by_cvss(query, args):
                                 (fn.COALESCE(CveMetadata.cvss3_score, CveMetadata.cvss2_score) <= args['cvss_to']))
     elif 'cvss_from' in args and args['cvss_from']:
         query = query.where(fn.COALESCE(CveMetadata.cvss3_score, CveMetadata.cvss2_score) >= args['cvss_from'])
-    elif 'cvss_to' in args and args['cvss_to']:
+    elif 'cvss_to' in args and (args['cvss_to'] or args['cvss_to'] == 0):
```

Comment:
I think conditions on line 45 and 38 have the same problem and some combinations with zeroes might not work.

Reply:
sure